src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,24 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
-);
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment }) => {
+  const { name, email, body } = comment;
+
+  return (
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">{name}</strong>
+        {' by '}
```

Comment:
it's better to use `span` instead
```suggestion
        <span> by </span>
```